app/presenters/hyrax/displays_image.rb

```diff
@@ -35,5 +41,10 @@ def iiif_endpoint(file_id)
           profile: Hyrax.config.iiif_image_compliance_level_uri
         )
       end
+
+      def unindexed_current_file_version
+        Rails.logger.warn "Indexed current_file_version for #{id} not found, falling back to Fedora."
```

Comment:
I couldn't find `current_file_version` defined anywhere.  Maybe something different could be used to make this warning clearer?